docs/network/alfajores/use-mobile-wallet.md

```diff
@@ -19,8 +19,6 @@ If you need more funds, you can always visit [faucet.celo.org](https://faucet.ce
 
```

Comment:
- this link is broken: (https://celo.org/build/wallet)
- this links to the first alfajores wallet: (https://play.google.com/store/apps/details?id=org.celo.mobile.alfajores)